### ISSUE TITLE
Waluigi's Taco Stand crashes when stomping a Goomba

### ISSUE THREAD

**Issue by anonymous7002** [project64/project64#1727]
I was playing Waluigi's Taco Stand, a SM64 rom hack.
When I stomped a goomba, the game crashed:
Fatal error: Stopped emulation

Many other players have experienced this bug.
Is it an emulator or a game bug? Because all other emulators work fine.

**Comment by GB-CG** [comment]
IIRC, this also occurs on a N64. It's a bug.

**Comment by oddMLan** [comment]
Try enabling Unaligned DMA under ROM settings

**Comment by anonymous7002** [comment]
It doesn't. There's a footage of the game running on N64.
It works fine on console.

**Comment by IronYoshi64** [comment]
Yeah some romhacks can be tricky. I got this game working perfectly using Jabo's Direct3D 1.7, 8MB for memory size, Counter Factor set to 2, and Advanced block linking set to Off. And in even newer builds, set Unaligned DMA to On. Pretty much every romhack needs that option on.

**Comment by oddMLan** [comment]
Disabling Advanced Block Linking fixes the issue
I'm thinking it's time to update the rdb with the latest entries from GoodN64 + other hacks that aren't in it (or will be: GoodN64 has been discontinued).
